tests/phpunit/tests/blocks/block.php

```diff
@@ -567,4 +567,65 @@ public function test_build_query_vars_from_query_block_page_with_offset() {
 			)
 		);
 	}
+
+	/**
+	 * @ticket 53257
```

Comment:
Should I change the ticket with the one introduced this function here as well? @gziolo

Reply:
Yes, that would be a better fit.